ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -1865,3 +1868,19 @@ enum pkcs11_rc add_missing_attribute_id(struct obj_attrs **pub_head,
 	else
 		return set_attribute(pub_head, PKCS11_CKA_ID, id2, id2_size);
 }
+
+enum pkcs11_rc get_key_data_to_wrap(struct pkcs11_object *obj, void **data,
+				    uint32_t *sz)
+{
+	switch (get_class(obj->attributes)) {
+	case PKCS11_CKO_SECRET_KEY:
+		if (get_attribute_ptr(obj->attributes, PKCS11_CKA_VALUE, data,
+				      sz))
+			return PKCS11_CKR_ARGUMENTS_BAD;
+		break;
+	default:
+		return PKCS11_CKR_GENERAL_ERROR;
```

Comment:
This should not happen, right? What about an `assert(0)`?

Reply:
Ok